docs/user-guide/zonal-average.ipynb

```diff
@@ -10,6 +10,21 @@
     "This section demonstrates how to perform Zonal Averaging using UXarray, covering both non-conservative and conservative methods.\n"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    lat_bands = np.arange(-90, 91, 30)
</pre><p> </p><p>Why this latitude band is generated here and why with a step of 30 is confusing to me, and I don't know if it is contributing to the content. It might get misleading for the expected outcome from the following sections.</p><p> </p><p>Maybe instead, either print the band for the default one in the beginning of the next section, or the one here at the section 2.4?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1403/discussion/#comment-2520082366'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>